meson-mode.el

```diff
@@ -935,19 +935,13 @@ Return the buffer containing the reference manual or nil."
     (recenter 0)
     (current-buffer)))
 
-(defun meson-lookup-doc-at-point ()
-  "Show Meson documentation related to current point.
-Currently, it shows something only for functions."
-  (interactive)
-  (if-let ((func (meson-function-at-point)))
```

Comment:
This removes some functionality that I was using. In the current version, if you type for example: `executable(` or `executable('xxx',` and press F1, the documentation for `executable` function will be shown. With your version, one has to move back to the executable and press F1 there. Can't you do something like calling meson-function-at-point if the identifier in meson-lookup-doc is empty?

Reply:
Fixed in 11cd2ba4fa2195de70ad2a3d193d29082980b4e0.